vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/DropdownSteps.java

```diff
@@ -104,180 +81,92 @@ public Select isDropDownWithNameFound(String dropDownName)
      * </tbody>
      * </table>
      */
-    @Then("a drop down with the name '$dropDownName' contains the items: $dropDownItems")
-    public void doesDropDownListContainItems(String dropDownName, ExamplesTable dropDownItems)
+    @Then("dropdown located by `$locator` contains options: $dropDownItems")
```

Comment:
`dropDownItems` -> `dropdownOptions`
the same for all variables and method names:
- `dropDown` -> `dropdown`
- `items` -> `options`